src/main/java/net/minecraftforge/fml/common/registry/FMLControlledNamespacedRegistry.java

```diff
@@ -935,4 +938,15 @@ public void remove()
         return new RegistryEvent.Register<I>(location, this);
     }
 
+    @Override
+    public void writeEntry(@Nonnull ByteBuf out, @Nonnull I entry)
+    {
+        ByteBufUtils.writeVarInt(out, getId(entry), 5);
+    }
+
+    @Override
+    public I readEntry(@Nonnull ByteBuf in)
```

Comment:
This should be marked `@Nullable` to match `FMLControlledNamespacedRegistry#getObjectById`